Source/WebCore/rendering/RenderLayoutState.h

```diff
@@ -111,6 +112,14 @@ class RenderLayoutState {
     void addLeadingTrimEnd(const RenderBlockFlow& targetInlineFormattingContext);
     void resetLeadingTrim() { m_leadingTrim = { }; }
 
+    void pushBlockStartTrimming(bool blockStartTrimming) { m_blockStartTrimming.append(blockStartTrimming); }
+    std::optional<bool> peekBlockStartTrimming() { return m_blockStartTrimming.isEmpty() ? std::nullopt : std::optional(m_blockStartTrimming.last()); }
```

Comment:
blockStartTrimming() is probably closer to WebKit style and also it's a const function.

Reply:
Changed